content/source/docs/cloud/run/modes-and-options.html.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: "cloud"
+page_title: "Run Modes and Options - Runs - Terraform Cloud and Terraform Enterprise"
+---
+
+# Run Modes and Options
+
+Terraform Cloud support many of the same modes and options available in local operations mode.
+
+## Destroy Mode
+
+[Destroy mode](/docs/cli/commands/plan.html#planning-modes) instructs Terraform to create a plan which destroys all objects, regardless of configuration changes.
+
+Command line users can activate destroy mode using `terraform plan -destroy` or `terraform destroy`, while API users can use the `is-destroy` option. Destroy runs may also be created through the UI via the workspace's "Destruction and Deletion" settings page.
+
+## Refresh-Only Mode
+
+-> **Version note:** Refresh-only support is available in Terraform CLI versions v0.15.4 and later, with server-side support available in Terraform Cloud and Terraform Enterprise v202106-1 and later.
```

Comment:
(For this and all notes like it)

"server-side" feels a little dated and emphasizing the remote version may be more helpful; how about:

```suggestion
-> **Version note:** Refresh-only support requires a Terraform Cloud workspace to use Terraform CLI version v0.15.4 or later. For Terraform Enterprise users, v202106-1 or later is required.
```

(There's the extra caveat that for the CLI workflow your _local_ version requires a minimum version, but I don't think that's as important here - if you're executing a command that isn't in the version of Terraform you're using locally, it's more obvious)

Reply:
This is a great point - I wordsmithed it just a bit more because I realized that Terraform Cloud support is kind of implied. How's this sound?

> -> **Version note:** Refresh-only support is available in Terraform Enterprise v202106-1 or later, and requires a workspace using at least Terraform CLI v0.15.2.